examples/api/java/StatisticsExample.java

```diff
@@ -0,0 +1,231 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Andres Noetzli
```

Comment:
Please call it `Statistics.java`, no other example has `Example` in the filename.

Reply:
Done. I renamed it to StatisticsExample.java since there is a class already named Statistics. 
The package name is used to avoid this conflict.